example/src/components/ChatboxWrapper.tsx

```diff
@@ -27,15 +26,16 @@ const ChatboxWrapper: FC<Props> = ({
   // use hooks
   const { data: currentMember } = hooks.useCurrentMember();
   const { data: chat } = hooks.useItemChat(chatId);
-  const memberships = hooks.useItemMemberships(chatId).data;
   // get chat messages
   const chatMessages = chat?.messages;
 
+  const { data: memberships } = hooks.useItemMemberships(chatId);
+
   const memberIds: string[] =
-    memberships?.map((m) => m.memberId)?.toArray() || [];
+    (memberships?.size && memberships?.map((m) => m.memberId)?.toArray()) || [];
```

Comment:
Did you really need the check on the size? Doesn't the map will be anyway empty and so result in an empty array?

Reply:
When i tested it was not behaving as expected. Adding the test on size made sure the `List` was non-empty (`.isEmpty()` did not work ...)